openwisp_notifications/websockets/handlers.py

```diff
@@ -1,7 +1,55 @@
 from asgiref.sync import async_to_sync
 from channels import layers
+from django.core.cache import cache
+from django.utils.timezone import now, timedelta
 
 from openwisp_notifications.api.serializers import NotFound, NotificationListSerializer
+from openwisp_notifications.utils import normalize_unread_count
+
+from .. import settings as app_settings
+from ..swapper import load_model
+
+Notification = load_model('Notification')
+
+
+def user_in_notification_storm(user):
+    """
+    A user is affected by notifications storm if it receives
+     - more than 6 notifications in 10 seconds
+     - more than 30 notifications in 60 seconds
```

Comment:
this docstring may have to be updated, I think it may be good to explain it in a more general way, since the defaults can be changed in the future and this docstring will remain outdated.